### PR TITLE
internal review

### PR DESCRIPTION
**Pull request by rata**

### CODE REVIEW

File: keps/sig-node/127-user-namespaces/README.md [hunk 1]
```diff
@@ -630,6 +630,8 @@ use container runtime versions that have the needed changes.
 
 - Gather and address feedback from the community
 - Be able to configure UID/GID ranges to use for pods
+- Add unit tests that excercise the feature gate switch (see section "Are there
```
Comment (giuseppe) [inline keps/sig-node/127-user-namespaces/README.md]: s/excercise/exercise/

File: keps/sig-node/127-user-namespaces/README.md [hunk 2]
```diff
@@ -670,6 +672,26 @@ enhancement:
   CRI or CNI may require updating that component before the kubelet.
 -->
 
+#### Kubelet and Kube-apiserver skew
+
+The apiserver and kubelet feature gate enablement are work fine in any combination:
```
Comment (giuseppe) [inline keps/sig-node/127-user-namespaces/README.md]: drop "are"

File: keps/sig-node/127-user-namespaces/README.md [hunk 3]
```diff
@@ -670,6 +672,26 @@ enhancement:
...
+
+1. If the apiserver has the feature gate enabled and the kubelet doesn't, then pod will show that
```
Comment (giuseppe) [inline keps/sig-node/127-user-namespaces/README.md]: "then the pod"

File: keps/sig-node/127-user-namespaces/README.md [hunk 4]
```diff
@@ -670,6 +672,26 @@ enhancement:
...
+   field and the kubelet will ignore it. Then, the pod is created without user namespaces.
+1. If the apiserver has the feature gate disabled and the kubelet enabled, the pod won't show this
```
Comment (giuseppe) [inline keps/sig-node/127-user-namespaces/README.md]: should this be "2."?

File: keps/sig-node/127-user-namespaces/README.md [hunk 5]
```diff
@@ -815,13 +840,13 @@ This section must be completed when targeting beta to a release.
 
 ###### How can a rollout or rollback fail? Can it impact already running workloads?
 
-The rollout is just a feature flag on the kubelet and the kube-apiserver.
+If one APIserver is upgraded while other's aren't and you are talking to a not
+upgraded the pod will be accepted (if the apiserver is >= 1.25).
```
Comment (giuseppe) [inline keps/sig-node/127-user-namespaces/README.md]: "to a not upgraded one, then then pod will be accepted (...)"

File: keps/sig-node/127-user-namespaces/README.md [hunk 6]
```diff
@@ -815,13 +840,13 @@ This section must be completed when targeting beta to a release.
...
-The rollout is just a feature flag on the kubelet and the kube-apiserver.
+If one APIserver is upgraded while other's aren't and you are talking to a not
+upgraded the pod will be accepted (if the apiserver is >= 1.25). 
 
-If one APIserver is upgraded while other's aren't and you are talking to a not upgraded the pod
-will be accepted (if the apiserver is >= 1.25). If it is scheduled to a node that the kubelet has
-the feature flag activated and the node meets the requirements to use user namespaces, then the
-pod will be created with the namespace. If it is scheduled to a node that has the feature disabled,
-it will be created without the user namespace.
+If it is scheduled to a node that the kubelet has the feature flag activated and
```
Comment (giuseppe) [inline keps/sig-node/127-user-namespaces/README.md]: should it be "to a node where the kubelet has the feature flag activated..."

File: keps/sig-node/127-user-namespaces/README.md (hunk 1 above)
Comment (rata) [reply keps/sig-node/127-user-namespaces/README.md]: Fixed, thanks!

File: keps/sig-node/127-user-namespaces/README.md (hunk 4 above)
Comment (rata) [reply keps/sig-node/127-user-namespaces/README.md]: Both work. The "trick" is that using always one, you can re-arrange and never care about it. Markdown, when rendered, shows the proper number :)

**Pull request merged**